recipes/notcurses/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+cmake_minimum_required(VERSION 2.8.12)
```

Comment:
```suggestion
cmake_minimum_required(VERSION 3.1)
```

Reply:
fixed, and actually synced to the project version, 3.14.0